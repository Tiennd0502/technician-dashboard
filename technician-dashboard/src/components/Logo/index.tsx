import { Box, Flex, Text } from '@chakra-ui/react';
import Image from 'next/image';

import { LOGO_URL } from '@/constants';

interface LogoProps {
  src?: string;
}

const Logo = ({ src = LOGO_URL }: LogoProps) => (
  <Flex gap='2.5'>
    <Image src={src} alt='Technician' width={37} height={38} />
    <Box>
      <Text
        width='fit-content'
        color='text.tertiary'
        fontWeight='normal'
        variant='textLg'
        letterSpacing='0.275px'
      >
        Technician
      </Text>
      <Text
        width='fit-content'
        fontWeight='normal'
        variant='textSm'
        color='text.tertiary'
        opacity='0.5'
        letterSpacing='0.193px'
      >
        At Home
      </Text>
    </Box>
  </Flex>
);

export default Logo;
