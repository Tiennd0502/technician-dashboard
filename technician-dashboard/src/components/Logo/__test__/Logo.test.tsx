import { render } from '@testing-library/react';

import Logo from '../index';

test('should render correctly', () => {
  const { container } = render(<Logo />);

  expect(container).toMatchSnapshot();
});
