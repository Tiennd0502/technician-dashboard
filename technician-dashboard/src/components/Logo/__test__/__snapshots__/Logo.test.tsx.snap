// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <div
    class="css-1owjrhv"
  >
    <img
      alt="Technician"
      class="chakra-image css-0"
      src="./technician.svg"
    />
    <div
      class="css-0"
    >
      <p
        class="chakra-text css-p3xeyl"
      >
        Technician
      </p>
      <p
        class="chakra-text css-uqf88n"
      >
        At Home
      </p>
    </div>
  </div>
</div>
`;
